packages/ui-react/src/components/Button/Button.module.scss

```diff
@@ -53,28 +56,31 @@ $large-button-height: 40px;
   &.default {
     color: currentColor;
     background-color: theme.$btn-default-bg;
+
+    &:focus {
+      @include accessibility.accessibleOutline;
+      outline-offset: 1px;
+    }
   }
 
   &.primary {
     color: var(--highlight-contrast-color, theme.$btn-primary-color);
     background-color: var(--highlight-color, theme.$btn-primary-bg);
 
     &:focus {
-      @include accessibility.accessibleOutlineContrast;
+      @include accessibility.accessibleOutline;
+      outline-offset: 1px;
```

Comment:
Since you've taken away the scaling, considering a11y, I have a minor doubt if 1px offset is enough for the primary buttons. The color of the button and the outline are the same, with only 1px between. Should we increase to 2px?

Example, current vs. accessibility.nl:

![Screenshot 2024-06-12 at 10 47 55](https://github.com/jwplayer/ott-web-app/assets/61868085/80784f8b-9368-4b23-907b-a21b3f40f7cc)

![Screenshot 2024-06-12 at 10 52 40](https://github.com/jwplayer/ott-web-app/assets/61868085/487ca522-e584-4e88-aee2-de8979ca7ebe)

Reply:
I have increased the outline offset for buttons to 2px, curious what others think

Reply:
Both look good

Reply:
The [WCAG guidelines](https://www.w3.org/TR/WCAG22/#focus-appearance) for Level AAA focus indicators (a level we’re not necessarily aiming for, but hey, it’s great if we can achieve it) state that the focus area should be at least as large as the area of a 2 CSS pixel thick perimeter. 

So, increasing the outline offset to `2px` here is the minimum requirement for that and thus a good change!